zenodo/modules/records/query.py

```diff
@@ -45,3 +49,4 @@ def search_factory(self, search, query_parser=None):
     """Search factory."""
     search, urlkwargs = _es_search_factory(self, search)
     return apply_version_filters(search, urlkwargs)
+    
```

Comment:
An empty line has been added or removed from the end of the file? Normally we prefer having only one empty line at the end of the file.